docs/configuration/tips.mdx

```diff
@@ -65,4 +65,4 @@ This configures the frequency the radio is set to.  Check out the [frequency cal
 - If you are part of a large mesh and don't know what a setting does, don't change it (unless you're super curious).
 - Leave your [MAX HOPS](/docs/configuration/radio/lora#max-hops) set to 3 unless you're sure you need more (or less) to reach your destination node.
 - TEST your settings and hardware before you install in hard-to-reach locations.
-- Connecting a node to the [public MQTT server](http://localhost:3000/docs/configuration/module/mqtt#connect-to-the-default-public-server) may publish the locations of all nodes in your mesh to the internet.  This will also add every globally connected node to your node database and potentially flood your mesh with all types of packets.
\ No newline at end of file
+- Connecting a node to the [public MQTT server](http://localhost:3000/docs/configuration/module/mqtt#connect-to-the-default-public-server) may publish the locations of all nodes in your mesh to the internet.  This will also add every globally connected node to your node database and potentially flood your mesh with all types of packets.
```

Comment:
looks like I made a mistake with this link too.  I believe it should be:
./module/mqtt#connect-to-the-default-public-server

Reply:
I'm not sure what you'd like the link to be changed to. Also I'm not sure what's going on with the diff here...I don't see any changes to this line

Reply:
More of a mental note.  I'll fix it later.  Not sure why it came up in the diff, but I'm glad it did!